packages/lit-dev-content/site/tutorials/content/reactivity/04.md

```diff
@@ -0,0 +1,9 @@
+One way your component can react to property changes is to compute values based on the new and existing property values.
+
+If you need to compute a value to render, you can compute it in the `render()` method itself. However a more general solution is to compute values in the `willUpdate()` callback. This has a couple of advantages:
+
+-   You can make computed properties available to [reactive controllers](/docs/composition/controllers/). The `willUpdate()` callback is called before the `hostUpdate()` callback is called on any [reactive controllers](/docs/composition/controllers/).
+
+-   If you subclass the component, you can supply a new `render()` method and still use  the computed properties.
+
+Updating a reactive property during `willUpdate()` does not trigger a new update cycle.
```

Comment:
It does not in render either. Let's either say this more fully or omit.

During the update cycle, setting any property will not trigger an additional update until the last callback in the cycle: `updated`.

Reply:
Clarified (I hope).